microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -21,43 +21,100 @@ This whitepaper focuses on the desktop version of Microsoft Edge. Parts of the d
 
-You can configure Microsoft Edge to collect diagnostic data about the address bar. Collected data includes the number of offered queries for all search providers. Go to `edge://settings/privacy`. In **Personalize your web experience** turn on the **Improve your web experience by allowing Microsoft to use your browsing history from this account for personalizing advertising, search, news, and other Microsoft services** setting.
+1. Click the **Search suggestions and filters** section.  The **Search suggestions and filters** settings page<!-- edge://settings/searchFilters --> opens.
 
-Typed characters and the websites you visit are stored locally on the device per profile.  To delete the data, go to `edge://settings/clearBrowserData`. In the **Clear browsing data** window, select the **Browsing history** checkbox, and select the **Clear now** button.
+1. Use the toggle **Show me search and site suggestions using my typed characters**:
 
-If Bing is your default search provider and you are signed in, you may delete your searches through the [Microsoft Privacy dashboard](https://account.microsoft.com/privacy/).  You can clear your browsing history and delete websites from appearing as suggestions in the address bar. Go to `edge://history`, and select **Clear browsing data**.  You can delete the data Microsoft collects from the address bar and search suggestions features on Windows 10 and later. Open **Start** > **Settings** > **Privacy** > **Diagnostics & feedback**. In **Delete diagnostic data**, select **Delete**.  All other data is deleted after 36 months.
+   ![The toggle in Edge Settings for "Show me search and site suggestions using my typed characters"](./index-images/search-suggestions-typed-chars.png)
 
-<!-- Deleting diagnostic data could use rewrite -->
+   If you turn off that toggle, your typed characters are no longer sent to your default search provider, but your search queries are still sent to your default search provider, to provide search results.<!--per ui: "If you turn this off, you'll only see suggestions from your favorites and history. Typed characters won't be sent to the search engine used in the address bar." -->
+
+
+<!-- ------------------------------ -->
+#### Microsoft Search in Bing
+
+If you're signed into Microsoft Edge using your work or school account (AAD account), your typed-in characters are also sent to Microsoft Search in Bing, which returns the work- or school-related content or cross-device suggested queries to you.  Your administrator can choose to turn on or off the work- or school-related content or query suggestions entirely for your organization, by using the `AddressBarMicrosoftSearchInBingProviderEnabled` policy.  If your administrator turns off that policy, the query suggestions and the results that are returned from your default search engine are shown in your present device's browser, but not in the browsers on your other devices.
```

Comment:
which do you prefer?
1. Your administrator can choose to turn on or off the work- or school-related content or query suggestions entirely for your organization, by using the AddressBarMicrosoftSearchInBingProviderEnabled policy.
2. Your administrator can choose to turn on or off the work- or school-related content or query suggestions entirely for your organization, by using the AddressBarMicrosoftSearchInBingProviderEnabled Microsoft Search in Bing policy.
3. Your administrator can choose to turn on or off the work- or school-related content or query suggestions entirely for your organization, by using the AddressBarMicrosoftSearchInBingProviderEnabled Microsoft Search in Bing (MSB) policy.

@dexterqu2024

Reply:
use: Microsoft Search in Bing (MSB) policy